src/Kdyby/Doctrine/Console/CommandHelper.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/**
+ * This file is part of the Kdyby (http://www.kdyby.org)
+ *
+ * Copyright (c) 2008 Filip Procházka (filip@prochazka.su)
+ *
+ * For the full copyright and license information, please view the file license.txt that was distributed with this source code.
+ */
+
+namespace Kdyby\Doctrine\Console;
+
+use Doctrine\DBAL\Tools\Console\Helper\ConnectionHelper;
+use Doctrine\ORM\Tools\Console\Helper\EntityManagerHelper;
+use Kdyby\Console\ContainerHelper;
+use Nette;
+
+
+
+/**
+ * @author Tomáš Jacík <tomas@jacik.cz>
+ */
+abstract class CommandHelper extends Nette\Object
+{
+
+	public static function setApplicationEntityManager(ContainerHelper $containerHelper, $emName)
+	{
+		/** @var \Kdyby\Doctrine\EntityManager $em */
+		$em = $containerHelper->getByType('Kdyby\Doctrine\Registry')->getManager($emName);
+		/** @var \Symfony\Component\Console\Helper\HelperSet $helperSet */
+		$helperSet = $containerHelper->getHelperSet();
+		$helperSet->set(new ConnectionHelper($em->getConnection()), 'db');
+		$helperSet->set(new ConnectionHelper($em->getConnection()), 'connection');
+		$helperSet->set(new EntityManagerHelper($em), 'em');
+		$helperSet->set(new EntityManagerHelper($em), 'entityManager');
```

Comment:
Not for doctrine commands to function. But if you look to helpers, there are connection and entityManger helper there. Shouldn't I update them all then?